tests/test_integration/conftest.py

```diff
@@ -1,6 +1,51 @@
+import os
+
+import pytest
+from dash.testing.composite import DashComposite
 from selenium.webdriver.chrome.options import Options
 
 
+class Helpers:
+    """A class to group helpful things when writing tests.
+
+    Added as a pytest fixture.
+
+    Example
+    -------
+    ```
+    def test_my_thing(helpers)
+        helpers.help_me()
+    ```
```

Comment:
love that there is an example ✨ 

minor comment: maybe `helpers.debug_screenshot` (since it exists)?